workspace_api/views.py

```diff
@@ -113,6 +127,39 @@ async def create_workspace(
     return {"name": workspace_name}
 
 
+def create_bucket_secret(workspace_name: str, credentials: Dict[str, Any]) -> None:
+
+    logger.info(f"Creating secret for namespace {workspace_name}")
+    k8s_client.CoreV1Api().create_namespaced_secret(
+        namespace=workspace_name,
+        body=k8s_client.V1Secret(
+            metadata=k8s_client.V1ObjectMeta(
+                name=CONTAINER_REGISTRY_SECRET_NAME,
+            ),
+            data={
+                "username": base64.b64encode(credentials["workspace_name"].encode()).decode(),
+                "password": base64.b64encode(
+                    credentials["password"].encode()
+                ).decode(),
```

Comment:
Those are the wrong fields, the actual fields of the secret can be found here: https://github.com/EOEPCA/rm-workspace-api/wiki/Workspace-representation-in-Kubernetes#k8s-secret-for-user-bucket